src/modules/numbers/numbers.service.ts

```diff
@@ -0,0 +1,58 @@
+import { Injectable, NotFoundException} from '@nestjs/common';
+import { CreateNumberDto } from './dto/create-number.dto';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { UkefNumber as Number } from './entities/number.entity';
+
+@Injectable()
+export class NumbersService {
+
+  constructor(
+    @InjectRepository(Number, "mssql-number-generator")
+      private readonly numberRepository: Repository<Number>,
+    ) {}
+
+  async create(createNumbers: CreateNumberDto[]): Promise<Number[]> {
+    let results = []
+    let ids = []
+    for (let i in createNumbers) {
```

Comment:
eslint seems to return this as a linting error. can we use `.map` instead?

Reply:
Refactored and added 2 TODO comments for future improvement.